### DISCUSSION TITLE
Milvus v2.2.X End of Support (Backward Compatible)

### DISCUSSION THREAD

**Discussion by gautam143751** [milvus-io/milvus#30567] [Q&A and General discussion]
Hello Community, 

I am trying to find information on whether Milvus releases will always be backward compatible i.e. how long the features rolled out against each version (major version) are supported by the community/product ?

Usually I find these details laid out in the release schedule/notes, but I couldn't find the information. I am referring to what NodeJS captures -  https://nodejs.org/en/about/previous-releases. 

Can we have something similar confirmed or published for Milvus going forward ? Thanks for all the work !

**Comment by yhmo** [comment]
Wait @xiaofan-luan to comment.

**Reply by xiaofan-luan** [reply]
We will still support Milvus 2.2 for a while, at least half an year, especially for the last few 2.2 releases, like 2.2.17.

2.2 will be end of support after 3.0 release, the target date release date could be mid of the year.

If you have new use cases, we recommened to use 2.3.8. 2.4 will be released soon this month but it won't be stable yet
